### ISSUE TITLE
BUG: Cannot select / copy relation names in UI

### ISSUE THREAD

**Issue by lingxi-at-contextual** [beekeeper-studio/beekeeper-studio#2489]
**Describe the bug**
In the table structure page > relations, we cannot select and copy relation names.

**To Reproduce**
1. Go to table structure (right click on table, "View Structure").
2. Go to "Relations" tab.
3. Cannot copy / select anything in the relations table.

**Expected behavior**
Should be able to select / copy anything from relations table.

**Screenshots / Gifs**

https://github.com/user-attachments/assets/0a16b605-4ca1-4fda-9282-595750c1a37b

**Version Information (please complete the following information):**
 - OS: macOS 15.1 Public Beta 3
 - App Version: 4.6.8
 - Database type and version: Postgres 15

**Comment by not-night-but** [comment]
Yeah we could definitely do with adding this to all tables in the UI

**Comment by RodrigoPerestrelo** [comment]
Hey! Can I get assigned to this issue?

**Comment by RodrigoPerestrelo** [comment]
Hey!
I have been working on the bug, and I created a method in the Vue files (TableRelations, TableIndexes, and TableTriggers) to handle double-click selection. I added this method to the respective columns (only those that made sense to me, avoiding fields such as checkboxes).
Can I apply this method to all columns in the Triggers Table?

I wanted to discuss these topics and get feedback so I can correct anything I may have done or thought incorrectly.

I did not find any tests regarding the UI to verify this. Is there a way to create some?

**Comment by not-night-but** [comment]
Hey! I would think you could apply that to all the columns in TableTriggers yeah. 

We don't currently have UI testing up, but we are working on getting a suite up and running with Playwright
